### PR TITLE
Sprint 6

### PR DESCRIPTION
**Pull request by harshp24asopalav**
This sprint covers the development of all use cases along with their frontend app.

### CODE REVIEW

File: diabetes.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#from tensorflow import keras
+
+
+class Diabetes:
+    def __init__(self, use_pretrained = True) -> None:
+        if use_pretrained:
+            print("Loading model...")
+            self.load_model()
+        else:
+            print("Training new model...")
```
Comment (enriqueespinosa) [inline diabetes.py:20]: Nice, clean code!

File: diabetes.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            self.load_data()
+            self.process_data()
+            self.model()
+            self.train_model()
+            self.save_model()
+            self.evaluate()
+
+    def load_data(self):
+        df = pd.read_csv('./data-collection/Diabetes/diabetes_prediction_dataset.csv')
```
Comment (enriqueespinosa) [inline diabetes.py:29]: This is not a secret, but it would be better not to hardcode it anymore.
There's many of these everywhere.

**Review by enriqueespinosa — CHANGES_REQUESTED** [review]
Docker secrets are not implemented.
I cannot find logger and log statements.
I am holding on the approval until I get clarification.

**Comment by enriqueespinosa** [comment]
I just received a new email with the same pull request. 
Nothing that I can see has changed in the past two hours since I last checked the code.
The expectation is that you develop and submit Docker Secrets and Log management.

**Review by ProfEspinosaAIML — APPROVED** [review]
Approved

**Pull request merged**